ci/travis-script.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+if [[ ${PLATFORM} == "Unix" ]]; then
+    mkdir -p build
+    cd build || exit 1
+
+    cmake -DPILIGHT_UNITTEST=1 \
+        -DCOVERALLS=ON \
+        -DCMAKE_BUILD_TYPE=Debug ..
+    make -j2
```

Comment:
Can you remove this dual core compilation?

Reply:
Will do